src/coreclr/tools/aot/ILCompiler.LLVM/Compiler/LLVMCodegenCompilation.cs

```diff
@@ -134,7 +134,15 @@ private unsafe void CompileSingleMethod(CorInfoImpl corInfo, LLVMMethodCodeNode
 
             try
             {
-                if (GetMethodIL(method).GetExceptionRegions().Length == 0)
+                var methodIL = GetMethodIL(method);
+
+                if (methodIL == null)
+                {
+                    methodCodeNodeNeedingCode.CompilationCompleted = true; //set to true here (*)
```

Comment:
```suggestion
                    methodCodeNodeNeedingCode.CompilationCompleted = true;
```
This comment does not make sense.